e2e/browserstack.json

```diff
@@ -21,7 +21,7 @@
     "project_name": "frontity/frontity",
     "build_name": "Latest Browsers - Local",
     "parallels": "1",
-    "exclude": ["wp-data/**/*"],
+    "exclude": ["wp-data/**/*", "project/**/*"],
```

Comment:
The BrowserStack workflow was failing because of the size of the [uploaded .zip file](https://github.com/frontity/frontity/runs/2346959011#step:8:173). Adding `project` to `exclude` makes the .zip file uploaded to BrowserStack much smaller (from ~64 MB to 4.3 MB), [fixing the problem](https://github.com/frontity/frontity/runs/2354901561#step:8:172).

All the tests [pass](https://github.com/frontity/frontity/runs/2354901561#step:8:675) now. :slightly_smiling_face: